pkgs/top-level/all-packages.nix

```diff
@@ -1450,13 +1450,13 @@ let
   # use config.packageOverrides if you prefer original gnupg1
   gnupg1 = gnupg1compat;
 
-  gnupg20 = callPackage ../tools/security/gnupg/20.nix {
+  gnupg20 = callPackage ../tools/security/gnupg/20.nix ({
     libgcrypt = libgcrypt_1_6;
-  };
+  } // (config.gnupg or {}));
```

Comment:
What's the way of using `config.pkg.option` if I'm using `~/.nixpkgs/config,nix` per user?